ntbs-service/Pages/Notifications/Edit/Episode.cshtml

```diff
@@ -1,40 +1,53 @@
 @page "{handler?}"
 @model ntbs_service.Pages_Notifications.EpisodeModel
 @using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+@using NHSUK.FrontEndLibrary.TagHelpers
 
 @{
-Layout = "../../Shared/_NotificationLayout.cshtml";
-ViewData["Title"] = "Notification - Hospital Details";
+    Layout = "../../Shared/_NotificationLayout.cshtml";
+    ViewData["Title"] = "Notification - Hospital Details";
+    var fullValidation = Model.Episode.ShouldValidateFull ? "True" : "False"; 
 }
 
 <h2>Hospital details</h1>
 
 <div>
     <div asp-validation-summary="ModelOnly" class="text-danger"></div>
 
-    <cascading-dropdowns inline-template>
+    <cascading-dropdowns model="Episode" shouldvalidatefull="@fullValidation" inline-template>
         <div>
-            <nhs-form-group nhs-form-group-type="Standard">
-                <label nhs-label-type="Standard" asp-for="Episode.TBServiceCode">
-                    Notifying TB service
-                </label>
-                <select nhs-select-type="Standard" asp-for="Episode.TBServiceCode" asp-items="Model.TBServices"  v-on:change="fetchHospitalList">
+            @{
+                var TBServiceCodeError = !Model.IsValid("Episode.TBServiceCode");
+                var TBServiceCodeGroupState = TBServiceCodeError ? Error : Standard;
+                var TBServiceCodeSelectErrorState = TBServiceCodeError ? SelectType.Error : SelectType.Standard; 
+            }  
+            <nhs-form-group ref="formGroupTBServiceCode" nhs-form-group-type=@TBServiceCodeGroupState> 
+                <label nhs-label-type="Standard" asp-for="Episode.TBServiceCode">Notifying TB service</label>
+                <span class="nhsuk-error-message" id="tb-service-error"
+                    ref="errorFieldTBServiceCode" asp-validation-for="Episode.TBServiceCode"></span>
+                <select ref="selectFieldTBServiceCode" nhs-select-type="@TBServiceCodeSelectErrorState" asp-for="Episode.TBServiceCode" 
+                    asp-items="Model.TBServices" v-on:change="(ev) => {fetchHospitalList(ev); validate('TBServiceCode')(ev);}">

```

Comment:
These arrow functions are definitely not OK, as they don't get compiled - but replacing them with normal functions should be trivial